plugins/channelrx/demodnavtex/navtexdemodsink.cpp

```diff
@@ -28,7 +28,6 @@
 #include "navtexdemodsink.h"
 
 NavtexDemodSink::NavtexDemodSink(NavtexDemod *packetDemod) :
-        m_navtexDemod(packetDemod),
```

Comment:
Remove from constructor parameters if not needed

Reply:
ok